docs/add_content_and_edit_views.md

```diff
@@ -0,0 +1,266 @@
+        _variables: function () {
+            var def = this.get('fieldDefinition');
+
+            return {
+                "isRequired": def.isRequired
+            };
+        },
+
+        /**
+         * Returns the input validity state object for the input generated by
+         * the tweet template
+         *
+         * See https://developer.mozilla.org/en-US/docs/Web/API/ValidityState
+         *
+         * @protected
+         * @method _getInputValidity
+         * @return {ValidityState}
+         */
+        _getInputValidity: function () {
+            return this.get('container').one('.ez-tweet-input-ui input').get('validity');
+        },
+
+        /**
+         * Returns the currently filled tweet value
+         *
+         * @protected
+         * @method _getFieldValue
+         * @return String
+         */
+        _getFieldValue: function () {
+            return this.get('container').one('.ez-tweet-input-ui input').get('value');
+        }
+    });
+
+    Y.eZ.FieldEditView.registerFieldEditView(
+        FIELDTYPE_IDENTIFIER, Y.eZ.TweetEditView
+    );
+});
+```
+
+### Registering the view
+
+We also have to create the YML file where we will tell the system to use our files. The example configuration looks like this:
+
+```yaml
+//Resources/config/yui.yml
+
+system:
+    default:
+        yui:
+            modules:
+                tweeteditview-ez-template:
+                    type: 'template'
+                    path: bundles/ezsystemstweetfieldtype/templates/fields/edit/tweet.hbt
+                ez-tweet-editview:
+                    requires: ['ez-fieldeditview', 'event-valuechange', 'tweeteditview-ez-template']
+                    dependencyOf: ['ez-contenteditformview']
+                    path: bundles/ezsystemstweetfieldtype/js/views/fields/ez-tweet-editview.js
+```
+
+We need to prepend this configuration, analogically like we did with our template. To do this, we have to edit dependency injection extension:
+
+```php
+public function prepend(ContainerBuilder $container)
+{
+    $configDirectoryPath = __DIR__.'/../Resources/config';
+
+    $this->prependYamlConfigFile($container, 'ez_platformui', $configDirectoryPath.'/yui.yml');
+    $this->prependYamlConfigFile($container, 'ezpublish', $configDirectoryPath.'/ez_field_templates.yml');
+}
+
+private function prependYamlConfigFile(ContainerBuilder $container, $extensionName, $configFilePath)
+{
+    $config = Yaml::parse(file_get_contents($configFilePath));
+    $container->prependExtensionConfig($extensionName, $config);
+}
+```
+
+Remember to dump your assets before trying to run this code for the first time.
+After this is done, we should be able to edit Content Item containing our Tweet Field Type. 
+
+## Implement content view
+
+The steps needed to create content view are analogical to the ones needed to create edit view.
+
+### Template file
+
+First, we need to add template file that will be responsible for displaying content of our Field Type:
```

Comment:
First, we need to add **a** template file that will be responsible for displaying **the** content of our Field Type: